site/components/policy/policy-box/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import React from 'react';
+
+import Statements from '../statements';
+import { H2 } from '../../text';
+
+import styles from './style.css';
+
+type StatementT = {
```

Comment:
Guessing 'T' stands for 'type'? Just wondering if this convention is neccesary as doesn't seem particularly common ?

Reply:
(in js / flow land anyway...)